rules/aws/persistence_ec2_security_group_configuration_change_detection.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2021/05/05"
+maturity = "production"
+updated_date = "2021/05/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Detects threat actors making configuration change to EC2 security groups. Threat actors may be able to maintain
+persistence, exfiltrate, and pivot in a AWS environment.
+"""
+false_positives = ["Unknown"]
```

Comment:
```suggestion
false_positives = [
    """
    A security group may be created by a system or network administrator. Verify whether the user identity, user
    agent, and/or hostname should be making changes in your environment. Security group creations from unfamiliar users
    or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
    """,
]```